HW_2/task_1.py

```diff
@@ -8,3 +8,11 @@
 0.56
 11
 """
+n = input('Введите вещественное число: ')
+
+def sum_digit_real_number(n):
+    return sum(map(int, list(str(n).replace('.', ''))))
```

Comment:
```suggestion
    return sum(map(int, list(n.replace('.', ''))))
```
`n` уже имеет тип данных `str`